pages/index.js

```diff
@@ -3,11 +3,29 @@ import styled from "styled-components";
 export default function Home() {
   return (
     <main>
-      <Heading>🐋Capstone Whales Template🐋</Heading>
+      <Heading>
+        <p>
+          Welcome to Tortuga app! To get started, simply tap the button in the
+          lower left corner to create your very first card.
+        </p>
+      </Heading>
     </main>
   );
 }
 
-const Heading = styled.h1`
-text-align: center;
+const Heading = styled.p`
```

Comment:
My personal preference: a `Heading` is a headline, like `h2`, `h3` etc. and not a paragraph.

Reply:
Also: with this code you are creating a `p` tag inside a `p` tag.
```
<p>
  <p>
    Welcome ...
  </p>
</p>
```